atlite/pv/irradiation.py

```diff
@@ -199,11 +201,20 @@ def clip(influx, influx_max):
 
         total_t = (direct_t + diffuse_t + ground_t).rename("total tilted")
 
+    if irradiation == "total":
+        result = total_t
+    elif irradiation == "direct":
+        result = direct_t
+    elif irradiation == "diffuse":
+        result = diffuse_t
+    elif irradiation == "ground":
+        result = ground_t
```

Comment:
The returned `xarray` objects should be named correctly, i.e. add
```
results = results.rename("total tilted")
```
and the respective other names.

We're also trying to consistently add `units` to the `xarray` objects whenever we touch them (in the past we did not provide units, which can be confusing). Can you add the appropriate unit to `results` as well?
Like e.g. here:
https://github.com/PyPSA/atlite/blob/40dd1fcc8f494379acd9f837e7ef1e0964cae3cd/atlite/pv/solar_position.py#L131

Reply:
Renamed and unit added to the `result`.